src/cmd/cli/command/compose.go

```diff
@@ -57,24 +88,26 @@ func makeComposeUpCmd() *cobra.Command {
 			const targetState = defangv1.ServiceState_DEPLOYMENT_COMPLETED
 			targetStateReached := false
 
-			go func() {
-				services := make([]string, len(deploy.Services))
-				for i, serviceInfo := range deploy.Services {
-					services[i] = serviceInfo.Service.Name
-				}
+			if !hasOnlyManagedResources {
```

Comment:
Maybe we keep this code here as is in case in the future we are able to return manage service statuses. It doesn’t hurt currently. Also nice that we control it on the backend so CLI will get messages once we add them.